### PR TITLE
Add Logs for Treatment/Transport Status

### PR DESCRIPTION
**Pull request by benn02**
### PR Checklist

Please make sure to fulfil the following conditions before marking this PR ready for review:

-   [ ] If this PR adds or changes features or fixes bugs, this has been added to the changelog
-   [ ] If this PR adds new actions or other ways to alter the state, [test scenarios](https://github.com/hpi-sam/digital-fuesim-manv-public-test-scenarios) have been added

### CODE REVIEW

File: shared/src/store/action-reducers/utils/log.ts [hunk 1]
```diff
@@ -64,6 +66,86 @@ export function logRadiogram(
...
+            `Der letzte Patient der Kategorie ${statusNames[patientStatus]} in allen Bereichen, die von der TO in ${simulatedRegion.name} verwaltet werden, ist abtransportiert worden`,
+            [
+                ...additionalTags,
+                createPatientStatusTag(state, patientStatus),
+                createSimulatedRegionTag(state, managingSimulatedRegionId),
+            ],
+            state.currentTime
+        )
+    );
+}
```
Comment (Greenscreen23) [inline shared/src/store/action-reducers/utils/log.ts:148]: Please merge these two function into one `logPatientTransport` function, that gets passed the description as a string

**Comment by Greenscreen23** [comment]
Otherwise lgtm

**Review by Greenscreen23 — CHANGES_REQUESTED** [review]
You should also create a changelog entry

File: shared/src/store/action-reducers/utils/log.ts (hunk 1 above)
Comment (benn02) [reply shared/src/store/action-reducers/utils/log.ts]: Done

**Comment by benn02** [comment]
> You should also create a changelog entry

Done

**Review by Greenscreen23 — APPROVED** [review]

**Pull request merged**